i18n/en/docusaurus-plugin-content-docs/current/about/understanding/naming.md

```diff
@@ -4,59 +4,47 @@ sidebar_position: 4
 
 # Naming
 
-## Problem
+Different developers have different experiences and contexts, which can lead to misunderstandings on the team when the same entities are called differently. For example:
 
-Due to the fact that each developer has his own experience and development context - [we are used to calling the same entities differently][disc-src], which can lead to misunderstandings within the team.
+- Components for display can be called "ui", "components", "ui-kit", "views", …
+- The code that is reused throughout the application can be called "core", "shared", "app", …
+- Business logic code can be called "store", "model", "state", …
 
-- *Components to be displayed - `ui` / `components` / `ui-kit` / `views` / ...*
-- *Code reusable in all parts of the application - the `core`/ `shared` / `app` / ...*
-- *Code business logic `store` / `model` / `state` / ...*
+## Naming in Feature-Sliced Design {#naming-in-fsd}
 
-## Naming in FSD
+The methodology uses specific terms such as:
 
-The methodology uses such terms as
+- "app", "process", "page", "feature", "entity", "shared" as layer names,
+- "ui', "model", "lib", "api", "config" as segment names.
 
-- `app`, `process`, `page`, `feature`, `entity`, `shared` - *[layers][refs-layers]*
-- `ui`, `model`, `lib`, `api` - *[segments][refs-segments]*
+Original naming is crucial to prevent confusion among team members and new developers joining the project. Using unique names also helps when seeking help from the community.
 
-Within the framework of the methodology, each of these terms has a [clear definition][refs-reference]
+## Naming Conflicts {#when-can-naming-interfere}
 
-When developing a project using the **Feature-Sliced Design** methodology, it is very important [to adhere to the original naming, in order to avoid misunderstandings][disc-src] both among the team members and outside of it.
+Naming conflicts can occur when terms used in the FSD methodology overlap with terms used in the business:
 
-- If a new developer comes to the project who is familiar with the methodology, **he should see the terms already familiar to him**
-- If you ask for help in the community, you will get an answer to your question faster, **if you use the same terminology**
+- `FSD#process` vs simulated process in an application,
+- `FSD#page` vs log page,
+- `FSD#model` vs car model.
 
-## When can naming interfere?
+For example, a developer seeing the word process in the code will spend unnecessary time trying to understand which process they are talking about.
```

Comment:
suggestion (non-blocking): for clarity, to distinguish between process, _the word_, and a process _of words_
```suggestion
For example, a developer seeing the word "process" in the code will spend unnecessary time trying to understand which process they are talking about.
```

Reply:
Also, I think that in this case, the developer in question is not talking about anything, they are reading someone else talking) so perhaps we could phrase it like this instead:
```suggestion
For example, a developer seeing the word "process" in the code will spend unnecessary time trying to understand which process is implied.
```